save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/PluginConfig.kt

```diff
@@ -110,14 +114,22 @@ data class GeneralConfig(
             suiteName,
             excludedTests ?: "",
             includedTests ?: "",
-            ignoreSaveComments ?: false
+            expectedWarningsPattern ?: defaultInputPattern,
         )
     }
 
     private fun errorMsgForRequireCheck(field: String) =
             """
-                Error: Couldn't find `$field` in [general] section of `$configLocation` config.
-                Current configuration: ${this.toString().substringAfter("(").substringBefore(")")}
-                Please provide it in this, or at least in one of the parent configs.
+                    Error: Couldn't find `$field` in [general] section of `$configLocation` config.
```

Comment:
There were something wrong with indents? `diktat` passed on this code previously

Reply:
this is how Idea fixes